setup.py

```diff
@@ -47,5 +47,7 @@
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5']
+        'Programming Language :: Python :: 3.5'],
+    entry_points={'console_scripts': 'qtpy = qtpy.cli:cli [cli]'},
```

Comment:
```suggestion
    entry_points={'console_scripts': 'qtpy = qtpy.__main__:main'},
```

To ensure our main entry point doesn't need to be updated if it changes

Reply:
This demands a whole turn around that includes some icky stuff I'd rather just avoid.  Not worth maybe having to update an entry point someday, maybe.

Reply:
I'm a little confused with how changing a few characters here and adding a couple lines of standard boilerplate to `__main__,py`, both of which I made for you as suggestions, "demands a whole turn around" in your approach. Furthermore, as explained above, could you explain what practical harms this "icky stuff" has in a trivial module containing no actual logic that outweigh the non-trivial risk and cost of maintaining two parallel entrypoints?